### PR TITLE
C525 Webcam Holder

### PR DESCRIPTION
**Pull request by LeoDova**
* [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?
A C525 webcam holder

#### How was it tested? 
Printed and shown in pictures

### CODE REVIEW

File: printer_mods/willikonk/webcam_holder/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# C525 webcam holder - Voron 2.4
+
+This is a webcam holder to attach the [Logitech C525](https://www.amazon.com/Logitech-Webcam-Portable-Calling-Autofocus/dp/B004WO8HQ4/ref=sr_1_3?crid=343YW4HG90NPC&keywords=c525&qid=1644847788&sprefix=c525%2Caps%2C288&sr=8-3)
```
Comment (cbc02009) [inline printer_mods/willikonk/webcam_holder/README.md:3]: remove ref and everything after it in the amazon link. it's tracking metadata.

**Review by cbc02009 — CHANGES_REQUESTED** [review]

**Comment by newphreak** [comment]
@LeoDova can you make the requested changes? :)

**Comment by MapleLeafMakers** [comment]
CAD Files need to be provided in STEP or SCAD format. you can include the .ipt file *in addition* to one of these.

**Comment by newphreak** [comment]
image
this should be chamfered so it doesn't need supports.

**Review by newphreak — APPROVED** [review]
Looks good to me

**Review by cbc02009 — APPROVED** [review]

**Comment by cbc02009** [comment]
@LeoDova this is good to go, but there are merge conflicts in the main readme.md file that need to be sorted out.

**Comment by newphreak** [comment]
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork
image

**Comment by FHeilmann** [comment]
Hi, 

I want to thank you for your submission and want to personally apologize that your submission has not gotten the review it deserves.
We are in the process to restructure the repo to streamline reviews and hopefully solve this issue in the future.

Unfortunately, this means that we will require you to resubmit the mod under the new system.

Please check [here](https://github.com/VoronDesign/VoronUsers/wiki/VoronUsers-Overhaul) for details and instructions on how to proceed. 
If you have any questions, please reach out in the #voronuser_mods channel on the VoronDesign discord.
If you have since moved on from this mod and don't want to resubmit, no further action is necessary from your side.

Bests,

Florian a.k.a. Haribro

**Pull request closed without merging**